src/services/transfer.ts

```diff
@@ -99,3 +99,21 @@ async function newCreateAccountTransaction(
 
   return createAccountTransaction
 }
+
+export const setImageSize = size => {
+  switch (size) {
+    case ImageSize.small:
+      return { image: 'is-32x32', font: 'is-size-6' }
+      break
+    case ImageSize.large:
+      return { image: 'is-128x128', font: 'is-size-1' }
+      break
+    case ImageSize.medium:
+    default:
+      return { image: 'is-64x64', font: 'is-size-3' }
+  }
+}
+
+export const validateAmount = (amount: string) => {
```

Comment:
these are pretty generic helpers, maybe they'd fit better in the `services/utils` module? there is already one called `renderAmount` there

Reply:
moved them